dashboard/src/components/ChartList/ChartList.tsx

```diff
@@ -41,7 +41,7 @@ class ChartList extends React.Component<IChartListProps, IChartListState> {
   public render() {
     const { charts: { isFetching }, pushSearchFilter } = this.props;
     const items = this.filteredCharts(this.props.charts.items, this.state.filter);
-    if (!isFetching && !items) {
+    if (!isFetching && (!items || items.length === 0)) {
```

Comment:
Is the !items required? items seems to be an array from the initial state https://github.com/kubeapps/kubeapps/blob/master/dashboard/src/reducers/charts.ts#L9